corehq/apps/app_manager/templates/app_manager/partials/releases/releases_table.html

```diff
@@ -34,6 +34,12 @@
         </span>
   </p>
   <p>
+    {% if request|toggle_enabled:"VIEW_APP_CHANGES" %}
+      <a class="btn btn-default" data-bind="attr: { href: hqImport('hqwebapp/js/initial_page_data').reverse('app_form_summary_diff', '{{ latest_build_id}}', '{{ app.id }}') }">
```

Comment:
Why not use a regular `{% url ... %}` here?

Reply:
:+1: